src/components/NFTPortfolio/SubscriptionForm.vue

```diff
@@ -82,7 +82,8 @@
       <Label preset="h4" align="center" class="mb-[8px]" :text="$t('portfolio_subscription_verify_title')" />
       <Label class="text-dark-gray" preset="p5" align="center">
         <i18n path="portfolio_subscription_verify">
-          <span class="font-[600] text-like-green underline" place="email">{{ walletEmailUnverified }}</span>
+          <span v-if="!unverifiedEmailLink" class="font-[600] text-like-green underline" place="email">{{ walletEmailUnverified }}</span>
+          <a v-else :href="unverifiedEmailLink" target="_blank" class="font-[600] text-like-green underline" place="email">{{ walletEmailUnverified }}</a>
```

Comment:
Prefer flipped `v-if` condition, i.e If we have `unverifiedEmailLink` (`v-if="!!unverifiedEmailLink"`) then we will use it in `href` (`:herf="unverifiedEmailLink"`)
Much easier to maintain later